### PR TITLE
test(server): kill validation schema mutation survivors

### PR DESCRIPTION
**Pull request by ng**
## Summary

- Add fresh-import tests for all 50 static validation-schema survivors reported by mutation run 30150630961.
- Pin regex anchors, accepted boundaries, defaults, enum members, and exact public validation messages.
- Reset the Vitest module cache before dynamic imports so module initialization executes while each Stryker mutant is active.

## Mutation result

Scoped validation-schemas.ts result: 128 killed, 0 survived, 0 no coverage, and 122 compile errors — 100% mutation score.

## Verification

- pnpm vitest run — 3,301 passed, 1 skipped
- pnpm tsc
- pnpm lint
- Drizzle schema validation through the pre-push hook
- Scoped Stryker run against src/server/validation-schemas.ts

Tracks #591.

Source run: https://github.com/sleepypod/core/actions/runs/30150630961

## Summary by CodeRabbit

* **Tests**
  * Added comprehensive validation coverage for primitive values, date-time formats, signal keys, enums, and automation rules.
  * Verified boundary conditions, default values, permitted ranges, list sizes, and user-facing validation messages.
  * Ensured test isolation so validation schema state doesn’t carry over between test cases.

<details open>
<summary>🛏️ <strong>Beam this PR onto a sleepypod</strong> ✨</summary>

Pick the snippet that matches your pod. **Already-installed pods can't use the curl bootstraps below** — sleepypod's egress firewall DROPs github.com, and the script that knows how to unblock WAN is the very file curl is trying to fetch. Use `sp-update` instead; it's on disk and opens WAN as its first step.

🔄 **Already on a sleepypod** (most common — review a PR on a running pod):
```bash
sudo sp-update test/mutation-run-30150630961
```

🚀 **Fresh pod / first install** — bootstraps from GitHub, rebuilds every push:
```bash
curl -fsSL https://raw.githubusercontent.com/sleepypod/core/test/mutation-run-30150630961/scripts/install \
  | sudo bash -s -- --branch test/mutation-run-30150630961
```

🎯 **Pin to this exact build** ([run 30397806373](https://github.com/sleepypod/core/actions/runs/30397806373) · `ca1bd6a`) — fresh-pod path, locked to one CI artifact for reproducible review:
```bash
curl -fsSL https://raw.githubusercontent.com/sleepypod/core/ca1bd6a999b70b8f71177af6ec35ec474a80f9af/scripts/install \
  | sudo bash -s -- \
      --branch test/mutation-run-30150630961 \
      --artifact-url 'https://nightly.link/sleepypod/core/actions/runs/30397806373/sleepypod-core.zip'
```

🧊 Artifact: [`sleepypod-core`](https://github.com/sleepypod/core/actions/runs/30397806373/artifacts/8703517822) · self-destructs in 30 days 💥
</details>

### CODE REVIEW

File: src/server/tests/validation-schemas.mutation.test.ts [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    }
+    for (const value of ['>', '>=', '<', '<=', '==', '!=']) {
+      expect(compareOpSchema.safeParse(value).success).toBe(true)
+    }
+    for (const value of ['+', '-', '*', '/']) {
+      expect(binaryOpSchema.safeParse(value).success).toBe(true)
+    }
+
+    expect(automationTriggerSchema.safeParse({ kind: 'tick', everyMin: 1 }).success).toBe(true)
+    expect(automationTriggerSchema.safeParse({ kind: 'tick', everyMin: 1440 }).success).toBe(true)
```
[the remark this answers is not in the document]
Comment (ng) [reply src/server/tests/validation-schemas.mutation.test.ts:107]: Addressed in ca1bd6a. The suite now rejects everyMin 0/1441, 281-character notifications, durationSec -1/86401, 121-character names, priority -1/1001, cooldown -1/1441, and 11 actions. The focused tests, typecheck, lint, repeated full suite, and a scoped 27-mutant replay all pass.
